.travis.yml

```diff
@@ -1,12 +1,27 @@
 language: node_js
+
 node_js: 10
+
 env:
   - NODE_ENV=ci
   - NODE_ENV=ci-betanet
     #  - NODE_ENV=betanet
+
 cache: yarn
-script:
-  - yarn lint
-  - yarn test
-  - yarn browserify && yarn run bundlewatch
 
+
+jobs:
+  include:
+    - name: yarn
+      script:
+      - yarn lint
+      - yarn test
+      - yarn browserify && yarn run bundlewatch
+
+    - name: fossa     
+      before_script:
+        - "curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/fossa-cli/master/install.sh | sudo bash"
```

Comment:
Piping `curl` into `sudo bash`! Fun times. Should this at least use a specific (vetted) commit, rather than the hot version from `master`? Seems like a security issue, even if it is only running in a CI environment.

Reply:
Pinned to a specific commit now than.